projects/go-lib/src/lib/components/go-loader/go-loader.component.ts

```diff
@@ -1,40 +1,33 @@
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, HostBinding, Input } from '@angular/core';
+import { fadeTemplateAnimation } from '../../animations/fade.animation';
 
 @Component({
   selector: 'go-loader',
   templateUrl: './go-loader.component.html',
-  styleUrls: ['./go-loader.component.scss']
+  styleUrls: ['./go-loader.component.scss'],
+  animations: [
+    fadeTemplateAnimation
+  ]
 })
-export class GoLoaderComponent implements OnInit {
-
-  @Input() loaderDone: boolean = false;
+export class GoLoaderComponent {
   @Input() loaderSize: string = 'medium';
   @Input() loaderType: string = 'neutral';
 
-  constructor() { }
-
-  ngOnInit() {
-  }
+  @HostBinding('@fadeTemplate')
+  public fadeTemplate: boolean = true;
```

Comment:
I don't _think_ we really need to assign a variable to this `@HostBinding`.

Reply:
Shrug, seems like it could be a nice override.